.gitignore

```diff
@@ -70,5 +70,8 @@
 /c++/m4/lt~obsolete.m4
 /c++/samples/addressbook
 
+#cmake artefacts
```

Comment:
Good CMake practice is to use out-of-source builds, so these lines are unnecessary (or should be added to your local gitignore/exclude file).